tests/src/test/java/ca/bc/gov/open/jag/efiling/page/PackageReviewPage.java

```diff
@@ -37,13 +37,18 @@
 
     public List<String> getPackageDetails() {
 
-        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("uncontrolled-tab-tabpane-documents")));
-        documentsTab.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@data-test-id='detailsTable']")));
 
         logger.info("Getting package details values");
 
         List<String> packageDetails = new ArrayList<>();
         packageValueDetails.forEach(webElement -> {
+            if (!webElement.isDisplayed()) {
+                logger.info("WebElement is still not visible. Waiting for all rows to be present: {}", webElement.isDisplayed());
+                wait.until(ExpectedConditions.visibilityOfAllElementsLocatedBy(By.xpath("//*[contains(@class,'bcgov-table-value')]/b")));

```

Comment:
Does this have a timeout?

Reply:
Yes, uses the default 30 second timeout specified in TestConfig class.